modules/bot/components/request/SystemMessageRequestHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace app\modules\bot\components\request;
+
+use TelegramBot\Api\Types\Update;
+
+class SystemMessageRequestHandler extends MessageRequestHandler
+{
+    public function getFrom(Update $update)
+    {
+        if ($message = $update->getMessage()) {
+            $from = $message->getFrom();
+        }
+
+        return $from ?? null;
+    }
+
+    public function getChat(Update $update)
+    {
+        if ($message = $update->getMessage()) {
+            $chat = $message->getChat();
+        }
+        return $chat ?? null;
+    }
```

Comment:
You don't need to override these functions because this class is inherited from `MessageRequestHandler` that already implements them